app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CsvUploadForm
+  include ActiveModel::Validations
```

Comment:
it's ok to include just the ActiveModel::Validations here but FYI we could also includes all other ActiveRecord helpers to this Form with `include ActiveModel::Model`

Reply:
yeah, my bad. Missed that line when reading the compass. adf505e4a0180d1abfdc0defb34ad7c441089a33